SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialRPCService.cpp

```diff
@@ -225,7 +250,20 @@ TArray<SpatialRPCService::UpdateToSend> SpatialRPCService::GetRPCsAndAcksToSend(
 		SpatialRPCService::UpdateToSend& UpdateToSend = UpdatesToSend.AddZeroed_GetRef();
 		UpdateToSend.EntityId = It.Key.EntityId;
 		UpdateToSend.Update.component_id = It.Key.ComponentId;
-		UpdateToSend.Update.schema_type = It.Value;
+		UpdateToSend.Update.schema_type = It.Value.Update;
+		if (It.Value.SpanIds.Num() > 1)
+		{
+			if (EventTracer != nullptr)
```

Comment:
This should be checked first (the `SpanIds.Num()` checks should only happen if EventTracer is there)